game/outfits.rpy

```diff
@@ -1,289 +1,2058 @@
-        store.JNTimeBlocks.mid_morning: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.late_morning: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.afternoon: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.evening: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.night: DEFAULT_OUTFIT_CASUAL_WEEKEND
+    _OUTFIT_SCHEDULE_WEEKEND_MEDIUM_AFFINITY = {
+        store.JNTimeBlocks.early_morning: get_outfit("jn_star_pajamas"),
+        store.JNTimeBlocks.mid_morning: get_outfit("jn_casual_clothes"),
+        store.JNTimeBlocks.late_morning: get_outfit("jn_casual_clothes"),
+        store.JNTimeBlocks.afternoon: get_outfit("jn_casual_clothes"),
+        store.JNTimeBlocks.evening: get_outfit("jn_casual_clothes"),
+        store.JNTimeBlocks.night: random.choice((get_outfit("jn_casual_clothes"), get_outfit("jn_hoodie_turtleneck")))
     }
 
-    DEFAULT_OUTFIT_SCHEDULE_WEEKDAY_LOW_AFFINITY = {
-        store.JNTimeBlocks.early_morning: DEFAULT_OUTFIT_UNIFORM,
-        store.JNTimeBlocks.mid_morning: DEFAULT_OUTFIT_UNIFORM,
-        store.JNTimeBlocks.late_morning: DEFAULT_OUTFIT_UNIFORM,
-        store.JNTimeBlocks.afternoon: DEFAULT_OUTFIT_UNIFORM,
-        store.JNTimeBlocks.evening: DEFAULT_OUTFIT_UNIFORM,
-        store.JNTimeBlocks.night: DEFAULT_OUTFIT_UNIFORM
+    _OUTFIT_SCHEDULE_WEEKDAY_LOW_AFFINITY = {
+        store.JNTimeBlocks.early_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.mid_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.late_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.afternoon: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.evening: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.night: get_outfit("jn_casual_clothes")
     }
 
-    DEFAULT_OUTFIT_SCHEDULE_WEEKEND_LOW_AFFINITY = {
-        store.JNTimeBlocks.early_morning: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.mid_morning: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.late_morning: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.afternoon: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.evening: DEFAULT_OUTFIT_CASUAL_WEEKEND,
-        store.JNTimeBlocks.night: DEFAULT_OUTFIT_CASUAL_WEEKEND
+    _OUTFIT_SCHEDULE_WEEKEND_LOW_AFFINITY = {
+        store.JNTimeBlocks.early_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.mid_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.late_morning: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.afternoon: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.evening: get_outfit("jn_school_uniform"),
+        store.JNTimeBlocks.night: get_outfit("jn_casual_clothes")
     }
 
-    def set_outfit(outfit):
-        """
-        Sets Natsuki's appearance using the given outfit.
+# Asking Natsuki to wear an outfit
+label outfits_wear_outfit:
+    if len(list(jn_outfits.get_all_outfits())) == 0:
```

Comment:
I think this `list` conversion should be in the method itself or just use `values` (the second is better as it's compatible with py3).
Also to make it more pythonic I suggest `if not jn_outfits.get_all_outfits():`

Reply:
Sure thing